CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+2.0.0 (unreleased)
+------------------
```

Comment:
please remove the header from the CHANGELOG file in dev. Adding the header is part of the release checklist.